media/css/foundation/annual-report-2019-2020.scss

```diff
@@ -436,6 +436,10 @@ $image-path: '/media/protocol/img';
     color: $color-black;
 }
 
+.mzp-c-modal-overlay-contents a:visited:hover, .mzp-c-modal-overlay-contents a:visited:focus, .mzp-c-modal-overlay-contents a:visited:active {
```

Comment:
This could be combined with the above rule and simplified a bit with some nesting:

```
.mzp-c-modal-overlay-contents {
    a:link,
    a:visited {
        color: $color-black;
        
        &:hover,
        &:focus,
        &:active {
            color: get-theme('link-color-hover');
        }
    }
}